app/assets/stylesheets/common.scss

```diff
@@ -546,6 +547,13 @@ body.compact {
     background: #fff;
     font-size: 12px;
 
+    > div {
+      position: relative;
+      float: left;
+      clear: both;
+      width: 100%;
+    }
+
```

Comment:
Oh gotcha - under `#sidebar` we have
* `div.search_forms`
* `div#flash`
* `div#browse_status`
* `div#sidebar_loader`  (hidden spinner)
* `div.welcome`
* `div#banner`

The `float:left; clear:both; width:100%;` makes it so that they all take up the full width of the sidebar and don't overlap.  `position:relative` is so that any absolutely positioned inner div (like `close-wrap`) gets positioned off of its parent div, rather than relative to the sidebar or some other nearby div.  It's useful for all toplevel divs under the sidebar to have this rule because they might not all be there (e.g. if user has dismissed the welcome div).